test/general/interface_check.pass.cpp

```diff
@@ -22,11 +22,14 @@
 #include <functional>
 #include <iterator>
 #include <vector>
+#if (defined(CL_SYCL_LANGUAGE_VERSION) || defined(SYCL_LANGUAGE_VERSION))
 #include <CL/sycl.hpp>
+#endif
 
 template <typename Policy, typename NewName>
 struct rebind_policy { using type = Policy; };
 
+#if ((defined(CL_SYCL_LANGUAGE_VERSION) || defined(SYCL_LANGUAGE_VERSION)) && ONEDPL_USE_DPCPP_BACKEND)
```

Comment:
Let's use a special macro for it, e.g. `TEST_DPCPP_BACKEND_PRESENT` or something like that. I'd suggest to define such macro in the test config we already have: `pstl_test_config.h`. It would be great if the file was also be renamed. Do you think it makes sense?

Reply:
Yes, that makes sense.  In the recent [RNG PR](https://github.com/oneapi-src/oneDPL/pull/75#discussion_r555729961) you proposed `test_config.h` and also adding a `SKIP_SYCL_TEST` macro.  Do we want to extend this PR to address that?

Reply:
Since we already touch this issue I'd suggest to extend PR with that, but without renaming (pstl_test_config -> test_config) since it requires changes in almost all test files

Reply:
Added `TEST_SYCL_PRESENT` and `TEST_DPCPP_BACKEND_PRESENT` macros. RNG tests are using the internal macro along with other algorithm tests.  I think updating that at the same time the config file is renamed since the files will be updated anyway would be best.

Reply:
ok